Scripts/Simulation_scripts/Energy_System.py

```diff
@@ -281,11 +305,14 @@ def save_simulation(self,simulation_name,filename=None):
         Function:
             Saves simulation outputs to a .csv file
         Inputs:
-            simulation_name     DataFrame output from Energy_System().simulation(...)
+            simulation_name     DataFrame output from Energy_System(**self.kwargs).simulation(...)
             filename            Name of .csv file to be saved as (defaults to timestamp)
         Outputs:
             Simulation saved to .csv file
         """
+        if not os.path.exists(self.simulation_storage):
```

Comment:
For this block, the `os` module provides a method that covers this case without needing an if statement:
```
os.makedirs(self.simulation_storage, exist_ok=True)
```
This will make the folder if it doesn't exist already, and won't throw an error, or overwrite the folder, if it already exists.